csi/controller_test.go

```diff
@@ -911,3 +911,222 @@ func TestControllerListVolumes(t *testing.T) {
 	}
 	assert.Equal(t, found, len(mockVolumeList))
 }
+
+func TestControllerCreateVolume(t *testing.T) {
```

Comment:
we could add some negative test cases for create volume

Reply:
Ah! I forgot. You got it.